src/clients/victoriaMetrics/tests/promMetricGauge.test.ts

```diff
@@ -0,0 +1,81 @@
+import { Gauge } from 'prom-client';
+import { promAgent } from '../promAgent';
+import {
+  GaugeConfig,
+  PromClientGaugeConfig,
+  PromMetricGauge,
+} from '../promMetricGauge';
+
+// Create mocks for promAgent and promClient.Counter
+promAgent.registerMetric = jest.fn();
+
+jest.mock('prom-client', () => ({
+  Gauge: jest.fn().mockImplementation(),
```

Comment:
You don't want to record/save this guy?

Reply:
nope, just testing the constructors been called